classes/Mock.php

```diff
@@ -137,7 +139,13 @@ public static function disableAll()
     public function call(array $arguments)
     {
         $this->recorder->record($arguments);
-        return call_user_func_array($this->function, $arguments);
+        
+        $Args = array();
```

Comment:
Could you please change that to lower case (`$args`)? I know that this is not part of PSR-2, but I'm so much used to the Java convention.